### PR TITLE
Remove module prefix

### PR DESCRIPTION
**Pull request by ben-albrecht**
* Removed redundant `Module:` in documentation, and all the tests expecting it.

* Also removed the (now useless) perl sed command from `fixInternalDocs.sh`

### CODE REVIEW

**Comment by ben-albrecht** [comment]
@benharsh volunteered to jump on this grenade

File: modules/internal/fixInternalDocs.sh [hunk 1]
```diff
@@ -62,7 +62,7 @@ function fixTitle() {
   # Module: ChapelFoo
   local titleLen=${#1}
   local header="$(printf '=%.0s' $(seq 1 $titleLen))"
```
Comment (benharsh) [inline modules/internal/fixInternalDocs.sh]: I think you can remove these two local vars and the comment above them, they will no longer be used.

**Comment by benharsh** [comment]
Looks good (once testing passes)!

File: modules/internal/fixInternalDocs.sh (hunk 1 above)
Comment (ben-albrecht) [inline modules/internal/fixInternalDocs.sh]: Aren't they still being used in `  perl -0777 -i -pe "s/$base\n=+\n/$1\n$header\n/g" $2 `?

**Comment by ben-albrecht** [comment]
Thanks for looking over - see the few more I caught in the last commit.

File: modules/internal/fixInternalDocs.sh (hunk 1 above)
Comment (benharsh) [inline modules/internal/fixInternalDocs.sh]: yup, my mistake

**Comment by ben-albrecht** [comment]
Will merge when paratest completes

**Pull request merged**